integrations/ragas/src/haystack_integrations/components/evaluators/ragas/metrics.py

```diff
@@ -123,27 +125,27 @@ class MetricDescriptor:
         Callable that converts the Ragas output format to our output format.
         Accepts a single output parameter and returns a list of results derived from it.
     :param init_parameters:
-        Additional parameters that need to be passed to the metric class during initialization.
+        Additional parameters that are allowed to be passed to the metric class during initialization.
     """
 
     metric: RagasMetric
     backend: Type[Metric]
-    input_validator: Callable[[Any], None]
     input_parameters: Dict[str, Type]
     input_converter: Callable[[Any], Iterable[Dict[str, str]]]
     output_converter: Callable[[Result, RagasMetric, Dict[str, Any]], List[MetricResult]]
-    init_parameters: Optional[Dict[str, Type[Any]]] = None
+    metric_params_validator: Callable[[RagasMetric, List[str], Dict[str, Any]], None]
```

Comment:
Since we no longer need to perform metric-specific validation of the parameters, let's just directly call the default function in the evaluator's `init_metric` method.

Reply:
Good point. Makes it much more simple again.